pkg/volume/csi/csi_client.go

```diff
@@ -687,6 +687,7 @@ func (c *csiDriverClient) NodeSupportsNodeExpand(ctx context.Context) (bool, err
 		for _, capability := range capabilities {
 			if capability.GetRpc().GetType() == csipbv1.NodeServiceCapability_RPC_EXPAND_VOLUME {
 				nodeExpandSet = true
+				break
```

Comment:
it might be better to just do `return true, nil` from here and remove rest of the code.

Reply:
Can you clarify a little bit: if we don't find match, we still need the return on line 693.

Reply:
sure but in that case `nodExpandSet` will already be false and we would be fine.